src/utils/retry_utils/src/retry_utils.c

```diff
@@ -259,5 +260,18 @@ void ReadRetryParamsArrayFromAgentConfigJson(
             value = DEFAULT_ENR_REQ_OP_MAX_JITTER_PERCENT;
         }
         params->maxJitterPercent = (double)value;
+
+        //
+        // Get the update polling interval retry setting, or default.
+        //
+        if (!ADUC_JSON_GetUnsignedIntegerField(
+                agentRetrySettingsJsonValue, SETTING_KEY_POLLING_INTERVAL_SECONDS, &context->pollingIntervalInSeconds))
+        {
+            Log_Warn(
+                infoFormatString, "", SETTING_KEY_POLLING_INTERVAL_SECONDS, ADUC_DEFAULT_POLLING_INTERVAL_SECONDS);
+
+            context->pollingIntervalInSeconds = ADUC_DEFAULT_POLLING_INTERVAL_SECONDS;
+        }
+        Log_Info("Read Update Polling Interval of %u from config", context->pollingIntervalInSeconds);
```

Comment:
nit: It makes it easier to search through the code base for traces if these are inline instead of using a const char* multiple times in different log statements.